sensors/depth_cam_LR.urdf.xacro

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0"?>
```

Comment:
I see you created copies of the depth cam xacro. Can we have only one xacro and use that single xacro for all 5 cameras?

Reply:
yes, I will delete it

Reply:
Thank you. If the simulation still works I merge :smile: